modules/test/playwright/tests/osb-faro-web/utils/navigation.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page} from '@playwright/test';
+
+import {faroConfig} from '../faro.config';
+
+export async function navigateTo(page: Page, pageName: string) {
+	await page.getByRole('link', {name: pageName}).first().click();
+}
+
+export async function navigateToACPage(page: Page) {
+	await page.goto(faroConfig.environment.baseUrl);
+
+	await page
+		.getByRole('link', {
+			name: 'FARO-DEV-liferay Liferay Demo Enterprise Plan',
+		})
+		.click();
+}
+
+export async function navigateToACPageViaURL(
```

Comment:
And them rename it to navigateToACSitesPage

Reply:
Oh ok, I think the `SitesPage` term is better, but I think we can continue with the `ViaURL` term. This term for me, identifies that we are accessing by directly loading the URL with the `projectID` and `channelID` and not accessing the workspace through the UI and etc. What do you think?

Suggestion:`navigateToACSitesPageViaURL`